### PR TITLE
Add Code'n'Color initiative

### PR DESCRIPTION
**Pull request by LLazarek**

### CODE REVIEW

File: _includes/navbar.html [hunk 1]
```diff
@@ -54,6 +54,7 @@
                               <a class="dropdown-item" href="cssi.html">CSSI</a>
                               <a class="dropdown-item" href="dbsec.html">DB-Sec</a>
                               <a class="dropdown-item" href="gwic.html">GWiC</a>
+                              <a class="dropdown-item" href="code-n-color.html">Code'n'Color</a>
```
Comment (Ettrai) [inline _includes/navbar.html:57]: This should go under the "Department Led" initiatives on line 65

File: code-n-color.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <p>
+      The support group meets weekly for coffee breaks and hosts BIPOC PhD holders who are leaders in the computing industry and academia. Advised and supported by BIPOC faculty in CS, ECE, and LS, the group is intended to provide a safe and supportive space for young scientists to interact with future faculty peers and climb the treacherous slopes of the ivory tower, cognizant of the unique, different demands and obstacles that BIPOC PhD students face.
+    </p>
+  </div>
+	<div class="col-md-4">
+    
+    <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
+      <div class="card-header">Information</div>
+      <div class="card-body">
+          <p class="card-text">Contact: josiah@northwestern.edu</p>
```
Comment (Ettrai) [inline code-n-color.html:35]: Maybe point of contact could serve this purpose better

File: _includes/navbar.html (hunk 1 above)
Comment (LLazarek) [reply _includes/navbar.html:57]: Oops! I'll fix it now.

File: code-n-color.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+  <div class="border-bottom col-md-12">  
+  </div>
+</div>
+
+<div class="row mt-2">
+  <div class="col-md-8">
+    <h2>Description</h2>
+    <p>
+      The support group meets weekly for coffee breaks and hosts BIPOC PhD holders who are leaders in the computing industry and academia. Advised and supported by BIPOC faculty in CS, ECE, and LS, the group is intended to provide a safe and supportive space for young scientists to interact with future faculty peers and climb the treacherous slopes of the ivory tower, cognizant of the unique, different demands and obstacles that BIPOC PhD students face.
```
Comment (Ettrai) [inline code-n-color.html:27]: Some line breaks could help formatting this paragraph

File: code-n-color.html (hunk 2 above)
Comment (LLazarek) [reply code-n-color.html:35]: Do you mean the title of the card?

File: code-n-color.html (hunk 3 above)
Comment (LLazarek) [reply code-n-color.html:27]: In the displayed paragraph or the source?

Comment (LLazarek) [reply code-n-color.html:27]: Hmm ok, I'll add a break between the sentences. But I think they envisioned it as a single paragraph?

Comment (Ettrai) [reply code-n-color.html:27]: Fair point

File: code-n-color.html [hunk 4]
```diff
@@ -24,15 +24,18 @@ <h4 class="text-secondary">PhD student support group for Black, Indigenous, and
...
+      Advised and supported by BIPOC faculty in CS, ECE, and LS, the group is intended to provide a safe and supportive space for young scientists to interact with future faculty peers and climb the treacherous slopes of the ivory tower, cognizant of the unique, different demands and obstacles that BIPOC PhD students face.
     </p>
   </div>
 	<div class="col-md-4">
     
     <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
       <div class="card-header">Information</div>
       <div class="card-body">
-          <p class="card-text">Contact: josiah@northwestern.edu</p>
+          <p class="card-text">Point of Contact: <a href="mailto:josiah@northwestern.edu">josiah@northwestern.edu</a></p>
```
Comment (Ettrai) [inline code-n-color.html:38]: Do you think this makes sense as it is or should it be something like: 
```
<p class="card-text">Point of Contact: Prof. Josiah Hester</p>          
<p class="card-text">Email: <a href="mailto:josiah@northwestern.edu">josiah@northwestern.edu</a></p>
```

File: code-n-color.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <p>
+      Advised and supported by BIPOC faculty in CS, ECE, and LS, the group is intended to provide a safe and supportive space for young scientists to interact with future faculty peers and climb the treacherous slopes of the ivory tower, cognizant of the unique, different demands and obstacles that BIPOC PhD students face.
+    </p>
+  </div>
+	<div class="col-md-4">
+    
+    <div class="card text-white bg-secondary mb-3" style="max-width: 20rem;">
+      <div class="card-header">Information</div>
+      <div class="card-body">
+          <p class="card-text">Point of Contact: <a href="mailto:josiah@northwestern.edu">josiah@northwestern.edu</a></p>
```
Comment (Ettrai) [inline code-n-color.html:38]: Do you think this makes sense as it is or should it be something like:

```
<p class="card-text">Point of Contact: Prof. Josiah Hester</p>          
<p class="card-text">Email: <a href="mailto:josiah@northwestern.edu">josiah@northwestern.edu</a>
```

File: initiatives.html [hunk 6]
```diff
@@ -42,9 +42,9 @@ <h2>Department Led</h2>
     
     <p>
       <ul>
+        <li><a href="code-n-color.html">Code'n'Color</a></li>
```
[the remark this answers is not in the document]
Comment (Ettrai) [reply initiatives.html:45]: This should go on line 47

**Pull request merged**